roles/cinder-data/tasks/main.yml

```diff
@@ -39,6 +39,10 @@
 - name: iscsi start
   service: name=open-iscsi state=started
 
+- include: vnx.yml
```

Comment:
this is starting to do more than just the navicli,  multipath and other stuff.   maybe we could call it `vnx_backend.yml` or something like that if `vnx` isn't descriptive enough.